portfolio/src/main/java/com/google/sps/util/SentimentAnalysisUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.util;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+
+public class SentimentAnalysisUtil {
+  public static float getSentimentScore(String text) {
+    float score = 0;
+
+    try {
+      Document document =
+          Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+      LanguageServiceClient languageService = LanguageServiceClient.create();
+      Sentiment sentiment = languageService.analyzeSentiment(document).getDocumentSentiment();
+      languageService.close();
+      score = sentiment.getScore();
+    } catch (IOException ioe) {
+      // Score remains as 0 if language service throws an IOException 
```

Comment:
what is the reason language service throwing IOException?

Reply:
Done. The IOException is thrown when the language service client cannot be instantiated. The docs do not provide any additional information about possible reasons for the exception.

I've included a screenshot of the documentation on the Google APIs site.

<img width="1102" alt="Screenshot 2020-06-22 at 11 39 58 AM" src="https://user-images.githubusercontent.com/37838247/85246433-34532f80-b47d-11ea-84c1-76b4ac0d2f57.png">

Reply:
since the exception is only at the language service client instantiation, lets move the instantiation to constructor, and enclose that part for exception only.

It's fine to change the function to as normal function instead of static, as if we are going to move the project to use dependency injection, we will do that anyway.

Reply:
I've made LanguageServiceClient an instance variable with Optional wrapped around it. The exception handling is now handled inside the constructor with a dependency injection into the CommentDatastore class.